### DISCUSSION TITLE
Midi to VST Host

### DISCUSSION THREAD

**Discussion by Orangery-DS** [SeanArchibald/scale-workshop#135] [Q&A]
With the MIDI button on in Scale Workshop, is it possible to output the information to a VST Host so that I can use different virtual synths and effects?  I tried it using VSTHost 2 by Hermann Seib but couldn't get it to work.

**Comment by vsicurella** [comment]
It is sort of other way around - you can use a MIDI controller to play within Scale Workshop instead of using the computer keyboard. Not all browsers support it though; this page can help with figuring that out: https://caniuse.com/midi

But it would be a very neat feature to output MIDI note & pitch bend data for use with synths that support MPE!

**Reply by Orangery-DS** [reply]
Oh that's a shame.  I use my computer keyboard for playing, as I prefer an isomorphic layout.  Thanks anyway.

**Comment by meszaros-lajos-gyorgy** [comment]
Added ticket to it and I'll do my best to add at least a basic support: https://github.com/SeanArchibald/scale-workshop/issues/143
